app/models/hackney/property.rb

```diff
@@ -92,4 +92,8 @@ def can_raise_a_repair?
   def is_tmo?
     letting_area.to_s.downcase.include? "tmo"
   end
+
+  def first_line_of_address
+    address.match?(/\s\s/) ? address[/.+(?<=\s{2})/] : address
```

Comment:
yesssss <3

Reply:
but what if the address is `1-50  pitcairn`? (note the double space before pitcairn)